TinyConstraints/Classes/TinyConstraints.swift

```diff
@@ -28,7 +28,7 @@
     import UIKit
 #endif
 
-public extension Constrainable {
+@MainActor public extension Constrainable {
```

Comment:
Not needed since `Constrainable` is already isolated to the `@MainActor`

```suggestion
public extension Constrainable {
```